RELEASE_NOTES.md

```diff
@@ -299,6 +299,39 @@ Our underlying SSH implementation has been updated to support the new ed25519 SS
 
 Chef Solo's `--delete-entire-chef-repo` option has been extended to work in Local Mode as well. Be warned that this flag does exactly what it states, and when used incorrectly, can result in loss of work.
 
+### sysctl now accepts a comments parameter
+
+The sysctl resource has been updated to allow additional (optional) comments. Comments can be passed as an array or a string. Any comments provided are prefixed with '#' signs and precede the sysctl setting in generated files.
```

Comment:
Formatting request: Please put tick marks on `sysctl` for both "`sysctl` resource" in the first sentence, and for "the `sysctl` setting" in the last sentence.